internal/loader/request_tracing.go

```diff
@@ -0,0 +1,78 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+package loader
+
+import (
+	acpv1 "azappconfig/provider/api/v1"
+	"context"
+	"fmt"
+	"net/http"
+	"os"
+	"strings"
+)
+
+type TracingKey string
+
+type RequestTracing struct {
+	IsStartUp bool
+}
+
+const (
+	RequestTracingKey     TracingKey = TracingKey("tracing")
+	AzureExtensionContext string     = "AZURE_EXTENSION_CONTEXT"
+)
+
+func createCorrelationContextHeader(ctx context.Context, provider acpv1.AzureAppConfigurationProvider, clientManager ClientManager) http.Header {
+	header := http.Header{}
+	output := make([]string, 0)
+
+	output = append(output, "Host=Kubernetes")
+
+	if tracing := ctx.Value(RequestTracingKey); tracing != nil {
+		if tracing.(RequestTracing).IsStartUp {
+			output = append(output, "RequestType=StartUp")
+		} else {
+			output = append(output, "RequestType=Watch")
+		}
+	}
+
+	if provider.Spec.Configuration.Refresh != nil &&
+		provider.Spec.Configuration.Refresh.Enabled {
+		output = append(output, "RefreshesKeyValue")

```

Comment:
Not needed. We don't have this in .NET either.

Reply:
I'm wondering why there is `RefreshesKeyVault` in dotnet provider but not `RefreshesKeyValue`?

Reply:
We know customers like dynamic config. We don't need to collect telemetry to know it.

I explained the reason for key vault refresh here https://github.com/Azure/AppConfiguration-KubernetesProvider/pull/47#discussion_r1645291492.